Seasons Defense/Assets/Scripts/SpawnManager.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public  class SpawnManager : MonoBehaviour
+{
+    private static Vector3[] _transformPositions;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        InitializePositions();
+    }
```

Comment:
Issue found with Null Object references at the beginning of the level. Let's change this to Awake to make sure the positions array is properly initialized before the first static method call.